src/util/check.cpp

```diff
@@ -2,9 +2,23 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include <util/check.h>
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
 
 #include <tinyformat.h>
+#include <util/check.h>
```

Comment:
nit: Usually we the header, which is directly related to the source file, at the top of the headers.

Reply:
yes, done